mk/def_rules.mk

```diff
@@ -6,9 +6,9 @@ ifndef COLOR_TTY
 COLOR_TTY := $(shell [ `tput colors` -gt 2 ] && echo true)
 endif
 
-LC_ALL := $(shell echo ${LC_ALL})
+LC_ALL := $(shell  /QOpenSys/pkgs/bin/python3.6  -c "import sys;print(sys.getdefaultencoding())")
```

Comment:
![image](https://user-images.githubusercontent.com/17009902/167517239-3570d90f-e28f-45ef-83c4-2f6753201de6.png)
Looks like we cannot use LC_ALL as the variable name here.